src/sections/ui/sanitized-html/SanitizedHtml.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import DOMPurify, { Config } from 'dompurify'
+
+interface SanitizedHTMLProps {
+  html: string
+  options?: Config
+}
+
+const defaultOptions: Config = {
+  ALLOWED_TAGS: ['b', 'i', 'em', 'strong', 'a'],
+  ALLOWED_ATTR: ['href']
+}
+
+export function SanitizedHTML({ html, options }: SanitizedHTMLProps): JSX.Element {
```

Comment:
We are not using this component anymore. We can remove this code, I mean, this work is already saved in the commits. Or if you prefer you can do what Guillermo suggested and branch off and keep this in a different branch in case we want to reuse this at some point.

Reply:
I agree, I will remove this since it is saved in a previous commit - I forgot to remove it when I made the new branch